docs/getting-started/introduction.mdx

```diff
@@ -16,7 +16,7 @@ import '@tldraw/tldraw/tldraw.css'
 
 export default function () {
 	return (
-		<div style={{ position: 'fixed', inset: 0 }}>
+		<div className="tldraw__editor">
 			<Tldraw />
 		</div>
 	)
```

Comment:
We also need to update the stackblits link on this page. We need to update the `examples` branch for that too.